pkg/csi/service/vanilla/controller.go

```diff
@@ -2642,9 +2643,18 @@ func (c *controller) processQueryResultsListVolumes(ctx context.Context, startin
 			nodeVMUUID, found := volumeIDToNodeUUIDMap[blockVolID]
 			if found {
 				volCounter += 1
-				//Populate csi.Volume info for the given volume
+				volumeId := blockVolID
+				migratedVolumePath, err := volumeMigrationService.GetVolumePathFromMigrationServiceCache(ctx, blockVolID)
+				if err != nil && err == common.ErrNotFound {
+					log.Debugf("volumeID: %v not found in migration service in-memory cache "+
+						"so it's not a migrated in-tree volume", blockVolID)
+					volumeId = blockVolID
```

Comment:
nit: This re-assignment can be removed, since it is already assigned in line: 2646

Reply:
Done